src/OpenSpaceToolkit/Astrodynamics/Trajectory.cpp

```diff
@@ -209,11 +218,11 @@ Trajectory Trajectory::GroundStrip(
         throw ostk::core::error::RuntimeError("LLA altitude must be zero.");
     }
 
-    const Velocity velocity = Velocity::MetersPerSecond({0.0, 0.0, 0.0}, aCelestial.accessFrame());
+    const Duration duration = anInstantArray.accessLast() - anInstantArray.accessFirst();
 
-    Array<State> states = Array<State>::Empty();
+    const Shared<Celestial> celestialSPtr = std::make_shared<Celestial>(aCelestial);
 
-    const Duration duration = anInstantArray.accessLast() - anInstantArray.accessFirst();
+    Array<physics::coordinate::Position> positions = Array<physics::coordinate::Position>::Empty();
```

Comment:
nit: Perhaps define the position properly with `using physics::coordinate::Position`? and apply to the rest of the file?

```suggestion
    Array<Position> positions = Array<Position>::Empty();
```

Reply:
We can't because `Position` is a named function in Trajectory :(